dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Register/PreferredNamePage.cshtml

```diff
@@ -22,11 +22,10 @@
                     <govuk-radios-item value="@true">Other
                         <govuk-radios-item-conditional>
                             <govuk-input asp-for="PreferredName" spellcheck="false">
-                                <govuk-input-label class="govuk-label--s"/>
+                                <govuk-input-label/>
```

Comment:
There still exist multiple input labels that are bold (including for inputs conditional on radio buttons)

Reply:
Can we remove the `<govuk-input-label/>` element altogether here?